src/Workspaces/CSharp/Portable/SyntaxDifferencing/CSharpSyntaxDifferenceService.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.SyntaxDifferencing;
+
+namespace Microsoft.CodeAnalysis.CSharp.SyntaxDifferencing
```

Comment:
I see your point.  Will have to think about that.  We can just state that this is a best effort diff and it might not be minimal.  As the API does not give guarantees, i think that's fine.